packages/react-composites/src/composites/MeetingComposite/MeetingComposite.test.tsx

```diff
@@ -36,7 +36,8 @@ function createMockCallAndChatAdapter(): CallAndChatAdapter {
         unparentedViews: []
```

Comment:
Not changing the file names yet?

Reply:
Leaving that as a separate PR to avoid bad git diffs in the changelist were if enough of the file changes git thinks there's a new file and a deleted file instead of a changed file.